src/binaries/query/main.rs

```diff
@@ -73,6 +76,17 @@ async fn main(_global_tracker: Arc<RuntimeTracker>) -> common_exception::Result<
     init_default_metrics_recorder();
     set_panic_hook();
 
+    let limits = get_own_limits().unwrap();
```

Comment:
We prefer to move this logic to a method(Such as set_open_file_limits()). If there are internal errors we can ignore them and allow the server to start.

Reply:
OK. what do you think `try_set_open_files_limit()` should be put? Can I just put it in main.rs in query package?